src/utils/string_parsing.ts

```diff
@@ -366,6 +366,31 @@ function guidToUuid(guid : Uint8Array) : Uint8Array {
   return uuid;
 }
 
+/**
+ * Decode string from bytes (UTF-8).
+ * Keeps reading until it reaches a byte that equals to zero.
+ * @param {Uint8Array} buffer
+ * @param {number} offset
+ * @returns {Object}
+ */
+function readTerminatedString(buffer: Uint8Array, offset: number): {
```

Comment:
Would be clearer as `readNullTerminatedString` as terminated string omit to tell us by what (https://en.wikipedia.org/wiki/Null-terminated_string)

Reply:
You're right !